include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //uint32_t           sgprAllocationGranularity;
+    //uint32_t           vgprsPerSimd;
+    //uint32_t           minVgprAllocation;
+    //uint32_t           maxVgprAllocation;
+    //uint32_t           vgprAllocationGranularity;
+
+    /* ShaderCoreProperties2AMD *//* provided by VK_AMD_shader_core_properties2 */
+    //VkShaderCorePropertiesFlagsAMD    shaderCoreFeatures;
+    //uint32_t                          activeComputeUnitCount;
+
+    /* BlendOperationAdvancedPropertiesEXT *//* provided by VK_EXT_blend_operation_advanced */
+    //uint32_t           advancedBlendMaxColorAttachments;
+    //VkBool32           advancedBlendIndependentBlend;
+    //VkBool32           advancedBlendNonPremultipliedSrcColor;
+    //VkBool32           advancedBlendNonPremultipliedDstColor;
+    //VkBool32           advancedBlendCorrelatedOverlap;
+    //VkBool32           advancedBlendAllOperations;
+            
+    /* ConservativeRasterizationPropertiesEXT *//* provided by VK_EXT_conservative_rasterization */
+    //float              primitiveOverestimationSize;
+    //float              maxExtraPrimitiveOverestimationSize;
+    //float              extraPrimitiveOverestimationSizeGranularity;
+    //VkBool32           primitiveUnderestimation;
+    //VkBool32           conservativePointAndLineRasterization;
+    //VkBool32           degenerateTrianglesRasterized;
+    //VkBool32           degenerateLinesRasterized;
+    //VkBool32           fullyCoveredFragmentShaderInputVariable;
+    //VkBool32           conservativeRasterizationPostDepthCoverage;
+            
+    /* CustomBorderColorPropertiesEXT *//* provided by VK_EXT_custom_border_color */
+    //uint32_t           maxCustomBorderColorSamplers;
+
+    /* DescriptorIndexingPropertiesEXT ---> MOVED TO Vulkan 1.2 Core  *//* provided by VK_AMD_shader_core_properties */
+
+    /* DiscardRectanglePropertiesEXT *//* provided by VK_EXT_discard_rectangles */
+    //uint32_t           maxDiscardRectangles;
+            
+    /* ExternalMemoryHostPropertiesEXT *//* provided by VK_EXT_external_memory_host */
+    //VkDeviceSize       minImportedHostPointerAlignment;
+    
+    /* FragmentDensityMapPropertiesEXT *//* provided by VK_EXT_fragment_density_map */
+    //VkExtent2D         minFragmentDensityTexelSize;
+    //VkExtent2D         maxFragmentDensityTexelSize;
+    //VkBool32           fragmentDensityInvocations;
+    
+    /* FragmentDensityMap2PropertiesEXT *//* provided by VK_EXT_fragment_density_map2 */
+    //VkBool32           subsampledLoads;
+    //VkBool32           subsampledCoarseReconstructionEarlyAccess;
+    //uint32_t           maxSubsampledArrayLayers;
+    //uint32_t           maxDescriptorSetSubsampledSamplers;
+    
+    /* GraphicsPipelineLibraryPropertiesEXT *//* provided by VK_EXT_graphics_pipeline_library */
+    //VkBool32           graphicsPipelineLibraryFastLinking;
+    //VkBool32           graphicsPipelineLibraryIndependentInterpolationDecoration;
```

Comment:
treat GL & GLES like a Vulkan device that does not advertise this extension, set everything to `false`

Reply:
`VK_EXT_graphics_pipeline_library` too much effort to expose, put as a TODO